argilla/src/argilla/records/_dataset_records.py

```diff
@@ -239,7 +241,7 @@ def log(
         created_or_updated = []
         records_updated = 0
         for batch in tqdm(
-            iterable=range(0, len(records), batch_size), desc="Adding and updating records", unit="batch"
+            iterable=range(0, len(records), batch_size), desc="2/2: Adding and updating records", unit="batch"
```

Comment:
```suggestion
            iterable=range(0, len(records), batch_size), desc="Adding and updating records", unit="batch"
```

Reply:
Not sure if 1/2 and 2/2 adds a lot or maybe makes it difficult to maintain?

Reply:
In fact, since step 1 is now completely local, we might as well skip the progress bar.